RCNN_tf.py

```diff
@@ -203,13 +163,12 @@ def convolutional_layer(input_data, num_input_chans, num_filter, filter_shape,
         output       (tf.Tensor): output tensor of the conv layer
     """
     conv_filter_shape = [filter_shape[0], filter_shape[1], num_input_chans,
-            num_filter]
+                         num_filter]
     strides = [1, stride[0], stride[1], 1]
 
     initializer = tf.contrib.layers.variance_scaling_initializer()
```

Comment:
Do we need to declare the initializer again over here? Can we not just declare it globally?

Reply:
Yea, you are right, I totally forgot that line. I will remove that!